generator/generator-opc-generator/generators/app/templates/src/_style.scss

```diff
@@ -0,0 +1,15 @@
+// Use this pre filled styles as you want
+// @import "../../opc-styles/_colors";
```

Comment:
@hybridx hey why are these commented?
also all of these can be imported in opc-styles.scss file, we dont have to add separate imports

Reply:
u can remove the .scss file from the index.js as u'll anyway need to import the scss file in this file to work with global variables

Reply:
The `.scss` file in the `index.ts` would be required as it is component specific. Removed all the other commented .scss and replaced with opc-styles